pkgs/applications/audio/zynaddsubfx/default.nix

```diff
@@ -123,8 +139,9 @@ in stdenv.mkDerivation rec {
       then "https://zynaddsubfx.sourceforge.io/zyn-fusion.html"
       else "https://zynaddsubfx.sourceforge.io";
 
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
     maintainers = with maintainers; [ goibhniu kira-bruneau ];
-    platforms = platforms.linux;
+    platforms = platforms.all;
+    broken = stdenv.isDarwin;
```

Comment:
Please add a comment why it is broken.

Reply:
I don't remember exactly (I started this a while ago in my NUR repo), so instead I'm just going to try to get it to build.